hazelcast/protocol/builtin.py

```diff
@@ -701,7 +702,7 @@ def decode(msg):
 
 class SqlPageCodec:
     @staticmethod
-    def decode(msg):
+    def decode(msg, to_object_fn):
         from hazelcast.sql import SqlColumnType, _SqlPage
```

Comment:
Is this import here in order to prevent circular imports?
If not, it would be better to move it to global in order to not to pay the (very small) cost.

Reply:
Yes, it was due to circular imports